README.md

```diff
@@ -1,2 +1,201 @@
-# client-java
-A java Grafeas api client
+# grafeas
+
+Grafeas API
+- API version: v1alpha1
+  - Build date: 2018-06-04T13:20:14.121-07:00
+
+An API to insert and retrieve annotations on cloud artifacts.
+
+
+*Automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen)*
+
+
+## Requirements
+
+Building the API client library requires:
+1. Java 1.7+
+2. Maven/Gradle
+
+## Installation
```

Comment:
These installation instructions were autogenerated. Do you know if these are accurate?